be/src/storage/rowset/beta_rowset_writer.cpp

```diff
@@ -279,7 +280,27 @@ Status BetaRowsetWriter::_final_merge() {
         }
     }
 
-    auto itr = new_aggregate_iterator(new_merge_iterator(seg_iterators), 0);
+    ChunkIteratorPtr itr = nullptr;
```

Comment:
Why do you add this logical? Please add comment to explain it.

Reply:
done